src/aat/resources/features/F-051/S-109.td.json

```diff
@@ -1,36 +1,54 @@
 {
-	"_guid_": "S-109",
-	"_extends_": "Base_Expecting_403",
-	"title": "must return appropriate negative response for a user not having a profile in CCD",
+  "_guid_": "S-109",
+  "title": "must return appropriate user setting from idam role for a user not having a profile in CCD",
+  "_extends_": "Base_Expecting_200",
+  "productName": "CCD Data Store",
+  "operationName": "Get default settings for user",
+  "method": "GET",
+  "uri": "/caseworkers/{uid}/profile",
+  "specs": [
+    "no profile in CCD",
+    "uses the Case ID of the case just created",
+    "has the 200 return code"
+  ],
+  "user": {
+    "username": "befta.caseworker.1.noprofile@gmail.com",
+    "password": "[[$CCD_BEFTA_CASEWORKER_1_PWD]]"
+  },
+  "request": {
+    "pathVariables": {
+      "uid": "[[DYNAMIC]]"
+    }
+  },
+  "expectedResponse": {
+    "body": {
+      "user": {
+        "idam": {
+          "id": "[[ANYTHING_PRESENT]]",
+          "email": "[[ANYTHING_PRESENT]]",
+          "forename": "[[ANYTHING_PRESENT]]",
+          "surname": "[[ANYTHING_PRESENT]]",
```

Comment:
These should also be consistent predefined values we can specifically expect in the response.